New version of Erizofc.js , throws an error on room.connect()

Please advice , new version of erizofc.js is throwing the following error on room.connect . 

`var Erizo = require('.erizofc').Erizo; ` ,  I don't think  `.Erizo` scope is used any more in the latest 
Erizofc.js 

`/home/ubuntu/opt/mvp/lib/erizofc.js:2212
          const token = __WEBPACK_IMPORTED_MODULE_5__utils_Base64__["a" /* default */ ].decodeBase64(spec.token);
                                                                                                         ^

TypeError: Cannot read property 'token' of undefined
    at Object.Room.that.connect (/home/ubuntu/opt/mvp/lib/erizofc.js:2212:106)
    at Stream.init (/home/ubuntu/opt/mvp/lib/app/stream.js:33:14)
    at /home/ubuntu/opt/mvp/lib/app/licode.js:100:28
    at XMLHttpRequest.p.onreadystatechange (/home/ubuntu/opt/mvp/lib/nuve.js:30:250)
    at l (/home/ubuntu/opt/mvp/lib/nuve.js:6:506)
    at IncomingMessage.<anonymous> (/home/ubuntu/opt/mvp/lib/nuve.js:10:372)
    at emitNone (events.js:110:20)
    at IncomingMessage.emit (events.js:207:7)
    at endReadableNT (_stream_readable.js:1045:12)
    at _combinedTickCallback (internal/process/next_tick.js:138:11)
    at process._tickCallback (internal/process/next_tick.js:180:9)
npm ERR! code ELIFECYCLE`

--- a/doc/client_api.md
+++ b/doc/client_api.md
@@ -899,17 +899,21 @@ You can configure and customize the way ErizoClient:
 
 Running erizo clients in your node.js applications.
 
-You can also run erizo clients in your node.js applications with the same API explained here. You can connect to rooms, publish and subscribe to streams and manage events. You need only to import the node module **erizofc.js**
+You can also run erizo clients in your node.js applications with the same API explained here. You can connect to rooms, publish and subscribe to streams and manage events. You need only to import the node module **erizofc.js**. This adds a new dependency that you will need to install: ` npm install socket.io-client`
 
 ```
-var Erizo = require('.erizofc').Erizo;
+var newIo = require('socket.io-client');
+var Erizo = require('.erizofc');
+```
+
+The line to initialize a room changes slightly. So, once you have a token:
+```
+var room = Erizo.Room(newIo, undefined, {token:'theToken'});
 ```
 
 And now you can use the API like explained for the browser case, calling `Erizo.Room`, `Erizo.Stream` and `Erizo.Events`. Note that you can not publish/subscribe streams with video and/or audio. We are working on this feature in order to develop another way of distribute video/audio streams.
 
 You can also use Erizo Client Logger for managing log levels, etc.
-
 ```
-var L = require('.erizofc').L;
-L.Logger.setLogLevel(2);
+Erizo.Logger.setLogLevel(Erizo.Logger.ERROR);
 ```

--- a/erizo_controller/erizoClient/src/ErizoFc.js
+++ b/erizo_controller/erizoClient/src/ErizoFc.js
@@ -1,7 +1,7 @@
 import Room from './Room';
 import { LicodeEvent, RoomEvent, StreamEvent } from './Events';
 import Stream from './Stream';
-
+import Logger from './utils/Logger';
 // Using script-loader to load global variables
 
 const Erizo = {
@@ -10,6 +10,7 @@ const Erizo = {
   RoomEvent,
   StreamEvent,
   Stream,
+  Logger,
 };
 
 export default Erizo;

